artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/view/ConsumerView.java

```diff
@@ -89,6 +90,8 @@ public Object getField(ServerConsumer consumer, String fieldName) {
             return consumer.getQueueName();
          case "queueType":
             return consumer.getQueueType();
+         case "filter":
+            return consumer.getFilter() == null ? "" : consumer.getFilter().getFilterString();
```

Comment:
Rather than the dupe logic here and on line 61, create a method to getFilterString.

Also if null, should return null, not empty string.
         return filter != null ? filter.getFilterString().toString() : null;

e.g. should be same logic as in QueueControlImpl